app/templates/gentelella/users/events/_table.html

```diff
@@ -72,15 +72,15 @@
                         {% if flag %}
                             {% for id, item in ticket_stats.iteritems() %}
                                 <div>
-                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span> 
+                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span>
                                 </div>
                                 <br>
                              {% endfor %}
                         {% else %}
                             {{ _("No Ticket Information") }}
                         {% endif %}
                     </td>
-                    <td><a href="{{ url }}" target="_blank">{{ url }}</a></td>
+                    <td style="width:200px;"><a href="{{ url }}" target="_blank">{{ url }}</a></td>
```

Comment:
<th style="width:150px;">{{ _("Public") }} URL</th>  @niranjan94  will doing this would work ? as I know little about js , checking jquery.dataTables.js  not getting where to make the change.